docs/installation/network-options.md

```diff
@@ -104,8 +110,27 @@ If you are using a custom cni plugin, i.e. a cni plugin different from flannel,
 --kubelet-arg "--node-ip=0.0.0.0" # If you want to prioritize IPv6 traffic, use "--node-ip=::" instead of "--node-ip=0.0.0.0".
 ```
 
-### IPv6 only installation
+## IPv6 only installation
 
 IPv6 only setup is supported on k3s v1.22 or above.
 
 > **Warning:** If your IPv6 default route is set by a router advertisement (RA), you will need to set `net.ipv6.conf.all.accept_ra = 2`; otherwise, the node will drop the default route once it expires. Be aware that accepting RAs could increase the risk of [man-in-the-middle attacks](https://github.com/kubernetes/kubernetes/issues/91507).
+
+## Distributed hybrid or multicloud cluster
+
+A k3s cluster can still be deployed on nodes which use different private networks and are not directly connected (e.g. nodes in different public clouds). To achieve this, k3s sets a mesh of tunnels that become a vpn mesh. These nodes must have have an assigned IP through which they can be reached (e.g. a public IP). The server traffic will use a websocket tunnel and the data-plane traffic will use a wireguard tunnel.
+
+To enable this type of deployment, you must add the following parameters in the server:
+```bash
+--node-external-ip $ServerExternalIP --flannel-backend wireguard-native --flannel-external-ip
```

Comment:
```suggestion
--node-external-ip <SERVER_EXTERNAL_IP> --flannel-backend wireguard-native --flannel-external-ip
```
Follows our standard "insert variable here" format we have in the rest of the docs.